src/lib.rs

```diff
@@ -409,7 +408,9 @@ impl<K: EnrKey> Enr<K> {
     pub fn signature(&self) -> &[u8] {
         &self.signature
     }
-
+    /// # Panics
+    ///
+    /// Will panic if the content does not has a public key
```

Comment:
this looks like a merge that left some traces. The panic info is already in the next lines